src/main/java/seedu/address/logic/parser/AssignCommandParser.java

```diff
@@ -32,19 +33,19 @@ public AssignCommand parseCommand(String args) throws ParseException {
         // Ensure no duplicate prefixes are provided
         argMultimap.verifyNoDuplicatePrefixesFor(ASSIGN_VOLUNTEER_PREFIX_NAME, ASSIGN_EVENT_PREFIX_NAME);
 
-        int volunteerId;
-        int eventId;
+        Index volunteerId;
+        Index eventId;
```

Comment:
can standardise naming to volunteerIndex and eventIndex too, else the rest LGTM!

Reply:
Alright it has been changed!